### DISCUSSION TITLE
Feature Request: NPM Package

### DISCUSSION THREAD

**Discussion by LukeMcCarthy-TW** [thoughtworks/talisman#390] [Ideas]
Hi Tailsman Team

I would like to request a feature to install tailsman via npm/yarn. This would allow anyone working on a node or react project to add tailsman as a dev dependency which would be installed with everything else. This would make it much easier to setup and run tailsman from a script inside the package.json file. It would also mean that team members don't need to have an extra step of installing tailsmain which is sometimes missed.

Possible user story
**As a:** Developer
**I want:** be able to install tailsman with npm/yarn
**So that:** Anyone working on the project will have tailsman installed without having to think about it and secrets won't be checked in

Thanks for all the hard work you put in.

Warm Regards
Luke

**Comment by svishwanath-tw** [comment]
@LukeMcCarthy-TW :  Someone has published an NPM package to wraps around the talisman CLI. https://www.npmjs.com/package/node-talisman.

Please close this issue if it works for you.
If not please update here about what is missing.
